api/php/services/RestaurantService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+include "php/Dbgateways/RestaurantDbGateway.php";
+
+class RestaurantService {
+    private $restaurantdbGateway;
+
+    public function __construct() {
+        $this->restaurantDbGateway = new RestaurantDbGateway();
+    }
+
+    public function processReview (string $restaurantName, string $restaurantAddress, string $restaurantPhone,
+        string $restaurantWebsite, string $review, int $deliciousnessScore, int $serviceScore, int $experienceScore, int $pricingScore, float $pricingValue,
+        array $images) {
```

Comment:
This method contains long parameter list which is a code smell. Perhaps we can think of a better way to do this with less parameters. Break the method into multiple methods, one method in restaurantService named getRestaurant which will take the parameters of the restauraunt and return the restaurant_id. Then we can have a "insertRestaurantReview" method in the ReviewDbGateway to insert the review into the table.